snippets/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+name: deployment
```

Comment:
How does the presence of these interact with the existing Helm snippets in `helm.json`?  They seem quite similar.